static/css/components/work.less

```diff
@@ -34,11 +40,127 @@ div.editionAbout {
     margin: 1em 0;
     font-size: 13px;
   }
+
+  p:first-child {
+    margin: 0;
+  }
+
 }
 
-.workDetails,
-.book-overview {
-  border-bottom: 1px solid @lightest-grey;
+.workFooter {
+  clear: both;
+  border-top: 1px solid @lighter-grey;
+  ul.listLists {
+    display: flex;
+    flex-wrap: wrap;
+  }
+  ul.listLists li {
+    flex: 1;
+    border: 0;
+  }
+}
+
+.book-description{
+  font-size: .9em;
+  margin-top: 5px;
+  margin-bottom: 20px;
+}
+
+.checkboxes label {
+display: inline-block;
+padding-right: 10px;
+white-space: nowrap;
+}
+
+.checkboxes input {
+vertical-align: middle;
+}
+
+.restricted-view{
+  display: block;
+  z-index: 1;
+}
+
+.restricted-height{
+  overflow: hidden;
+  height: 50px;
+  background-image: linear-gradient(to bottom, transparent, white);
```

Comment:
This won't work because it'll be behind the text (whose background is already white anyways. Delete it. (Or experiment with something like `mask-image: linear-gradient(to bottom,black,transparent)`)

Reply:
```suggestion
```